src/bloodhound.yml

```
@@ -20,10 +20,48 @@
 
     # BloodHound will not work unless you change the Neo4j password
     # from the default value.
+    #
+    # Note that the version of the official Neo4j package includes an
+    # epoch prefix in the version, followed by a colon character.
+    # This can be seen here:
+    # https://debian.neo4j.com/
+    #
+    # If we replace the colon by an exclamation point, then the
+    # resulting version is consistent with the pep440 version_type
+    # parameter, as described in these links:
+    # https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_tests.html#comparing-versions
+    # https://peps.python.org/pep-0440/
     - name: Set Neo4j password
-      ansible.builtin.command:
-        cmd: /usr/bin/neo4j-admin set-initial-password {{ password }}
-      changed_when: false
+      block:
+        - name: Gather the package facts
+          ansible.builtin.package_facts:
+
+        - name: Get version of the installed Neo4j package
+          ansible.builtin.set_fact:
+            installed_neo4j_version: >-
+              {{ ansible_facts.packages['neo4j'] |
+              map(attribute='version') | first |
+              replace(':', '!') | replace('"', '') }}
+
+        - name: Set neo4j password on version < 5
```

If we're capitalizing this elsewhere.

```suggestion
        - name: Set Neo4j password on version < 5
```